### DISCUSSION TITLE
no suitable node IP found , please make sure .... 3 control planes setup

### DISCUSSION THREAD

**Discussion by matjazantloga** [siderolabs/talos#9185] [Q&A] [answered]
when looking at logs i found following,

is this the reason why there is no IP for host?

![image](https://github.com/user-attachments/assets/7e52c458-1c88-49e6-bb25-6806e252946a)

there should not be any conflict according to config
![image](https://github.com/user-attachments/assets/fcb50e5b-c521-496d-b5bb-e7c2a712b649)

ips for nodes are from dhcp in range 10.99.100.132-134

**Comment by smira** [comment, marked as answer]
The default service subnet you're using [includes your machine IPs](https://www.talos.dev/v1.7/introduction/troubleshooting/#conflict-on-kubernetes-and-host-subnets).

![image](https://github.com/user-attachments/assets/d6e30d05-cad2-4ea4-bf90-951d52139a3a)

**Comment by matjazantloga** [comment]
aaaaa , its 12 bit mask .... 🙂


Best regards, s prijaznimi pozdravi

Matjaž Antloga, Ortoscale d.o.o.

________________________________
